library/src/main/java/com/novoda/simplechromecustomtabs/SimpleChromeCustomTabs.java

```diff
@@ -20,42 +19,35 @@
 
 public final class SimpleChromeCustomTabs implements WebNavigator, Connection, AvailableAppProvider {
 
-    private static Context applicationContext;
     private Connection connection;
     private WebNavigator webNavigator;
     private AvailableAppProvider availableAppProvider;
 
-    private SimpleChromeCustomTabs() {
-        //no-op
-    }
-
-    private static class LazyHolder {
-        private static final SimpleChromeCustomTabs INSTANCE = new SimpleChromeCustomTabs();
+    /**
+     * @deprecated There is no need to call it anymore in order to use this library,
```

Comment:
every decent sized app in the world extends Application.

most libraries need some type of upfront initialisation, keeping this is a sensible architecture choice for a public facing library, to avoid breaking api changes.

Happy to go with a concencus if it is wanted to be removed, but you need to understand why I would want to keep it :-)